data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,743 @@
+
+include "scripting/richtext_scenarios.lua"
+
+function jundlina(title, text)
+   return speech("map:princess.png", "2F9131", title, text)
+end
+function loftomor(text)
+   return speech("map:loftomor.png", "FDD53D", "Loftomor", text)
+end
+function sidolus(text)
+   return speech("map:sidolus.png", "FF1A30", "Sidolus", text)
+end
+function colionder(text)
+   return speech("map:colionder.png", "33A9FD", "Colionder", text)
+end
+function opol(text)
+   return speech("map:opol.png", "FF3FFB", "Opol", text)
+end
+function ostur(text)
+   return speech("map:ostur.png", "375FFC", "Ostur", text)
+end
+function kalitath(text)
+   return speech("map:kalitath.png", "FF0000", "Kalitath", text)
+end
+
+
+-- =======================================================================
+--                                Objectives
+-- =======================================================================
+
+obj_find_port = {
+   name = "obj_find_port",
+   title = _("Find a place to build a port"),
+   number = 1,
+   body = objective_text(_("Find Portspace"),
+   p(_([[To set foot on this island, we must first find a building space where a port can be built. Explore the nearby coast line to find such a building plot.]])) ..
+   p(_([[After the port is built we should cancel the other ship’s expedition, to unload its wares as well.]])))
+}
+
+obj_basic_infrastructure = {
+   name = "obj_basic_infrastructure",
+   title = _("Ensure the supply of build wares"),
+   number = 4,
+   body = objective_text(_("Establish Basic Production Chain"),
+   p(_([[We will need at least a quarry, a few woodcutters, and a sawmill. A forester might or might not be needed depending on how fast the trees are growing around us. Find the balance.]])))
+}
+
+obj_tools = {
+   name = "obj_tools",
+   title = _("Build tools quickly"),
+   number = 1,
+   body = objective_text(_("Make New Tools"),
+      p(_([[Most of our tools have been lost in the ship tragedy. Our people will not be able to fulfill their duties without proper tools. Luckily we still have some iron ore and plenty of coal. Build a production of tools, as it is a prerequisite for success on the island. Don’t forget to build a smelting works as well.]])))
+}
+
+obj_mining = {
+   name = "obj_mining",
+   title = _("Search the mountains for resources"),
+   number = 3,
+   body = objective_text(_("Build Mines"),
+      p(_([[As we are able to make tools again, our supply of mining resources needs to be replenished soon. Explore the mountains and build at least one mine of each type to mine coal, iron and gold.]])))
+}
+
+obj_explore = {
+   name = "obj_explore",
+   title = _("Explore the surroundings"),
+   number = 1,
+   body = objective_text(_("Explore Your Surroundings"),
+   p(_([[We are fairly sure that we are not alone on the island. We do not know if the others on this island are friends or foes, so we need to tread carefully. Explore the island by building at least one habitat for a scout.]])))
```

Comment:
```suggestion
   p(_([[We are fairly sure that we are not alone on the island. We do not know if the others on this island are friends or foes, so we need to tread carefully. Explore the island by building at least one house for a scout.]])))
```
Exhibit number 29, *Scoutus communis ssp. atlanteus*… Habitats are for animals ;)